packages/gatsby/src/utils/api-runner-error-parser.js

```diff
@@ -36,6 +36,20 @@ const errorParser = ({ err }) => {
       break
     }
   }
+    
+  if (!structured) {
+    return {
+      id: `11321`,
+      context: {
+        message: err instanceof Error
```

Comment:
nested ternary's are a nightmare. Could we simplify this to `err?.message ?? err ?? undefined` is that equivalent?

Reply:
We could. But in light of https://github.com/gatsbyjs/gatsby/pull/20597, I think it'd be better to type it and to consider `err` an `unknown` and check its type until the return type is what we expect.

I agree we could do without the nested ternary tho

Reply:
I might actually open a PR to #20597 to do that